src/Language/TypeScriptFlow.ts

```diff
@@ -154,8 +157,22 @@ export abstract class TypeScriptFlowBaseRenderer extends JavaScriptRenderer {
         return "function cast<T>(obj: any, typ: any): T";
     }
 
-    protected get typeAnnotations(): { any: string; anyArray: string; string: string; boolean: string } {
-        return { any: ": any", anyArray: ": any[]", string: ": string", boolean: ": boolean" };
+    protected get typeAnnotations(): {
+        any: string;
+        anyArray: string;
+        anyMap: string;
+        string: string;
+        stringArray: string;
+        boolean: string;
+    } {
+        return {
+            any: ": any",
+            anyArray: ": any[]",
+            anyMap: ": {[k: string]: any}",
```

Comment:
I usually see this written as `{ [key: string]: any }`, with spaces inside the `{ }`

Reply:
Yes.